### ISSUE TITLE
Probable bug: can't enable Share Lending in auto investing accounts if you didn't check the box during the initial account application

### ISSUE THREAD

**Issue by ElectricRCAircraftGuy** [ElectricRCAircraftGuy/bug_reports#17]
Under the "More" section of an auto investing account, there should be a section and slider to enable/disable Share Lending. If, during the application process, however, you did _not_ check the box to approve Share Lending, then you can _never_ activate this feature later. That's the bug. There should be a way to activate it after opening the account. 

I just called SoFi and the rep said it was active on my account, and would probably show that button in the next 24 hours or so, but it's not showing up now. I need to confirm this bug. If this is _not_ the case that it shows up later, then the bug is confirmed and I need to screenshot the whole account opening process and everything too, to show how to replicate it. 

I suspect that the bug _does_ exist, and that I'm going to have to *close* this entire investment account and *reopen a new one* just to get Share Lending turned on. This would be really dumb.

**Comment by ElectricRCAircraftGuy** [comment]
On a computer, under a Joint Automated investing account, I can click More --> Share lending at the left, and here's what I see: an option to turn it on and off:

![image](https://github.com/ElectricRCAircraftGuy/bug_reports/assets/6842199/f52d4a5f-2c19-4fdf-8f21-506203fc580e)

On my account I Just opened, I *can* see this too! It's been 8 days, and I don't know when this got populated in my account. But, my report above is *not* a bug, as shown by the fact this option _did_ show up on this account at some point, so I can close this issue.

---

But, there is a different problem: the $6 I put in there over the last 2 weeks hasn't resulted in any automated trades to buy into the stock market. Why? It should have.

**Comment by ElectricRCAircraftGuy** [comment]
I called SoFi, and spoke to Joe again (same guy, by chance, who I spoke to when I first opened this issue). He had to turn on a **"Rebalance" feature** on his end for this account. He thinks he may have overlooked it since he had to manually approve the opening of my account for me before after verifying proof of address documents. 

This **"rebalance" feature** needs to be turned *on* for automatic rebalancing ETF purchases based on your risk selection in these automated accounts. So, now that it's on, I should see my $6 get automatically purchased into the appropriate "aggressive" strategy ETFs over the next 1 to 2 business days (ex: by Monday or Tuesday, since today is Friday). So, that's done.

Note: the purpose of them being able to turn _off_ this "rebalance" feature manually is to prevent repurchases into the market when someone is trying to fully cash out and close an account or something.
